### ISSUE TITLE
Missing documentation about removal of font-awesome-sprockets

### ISSUE THREAD

**Issue by dacook** [FortAwesome/font-awesome-sass#207]
In the upgrade from 5 to 6, I found that this import fails:

     @import 'font-awesome-sprockets';

The [FA 5 readme](https://github.com/FortAwesome/font-awesome-sass/tree/5.15.1#a-ruby-on-rails) says it's required, but the [FA 6 readme](https://github.com/FortAwesome/font-awesome-sass/tree/6.1.0) omits it, so presumably it's no longer required.

As you can see on the above links, there used to be a second with details about upgrading, but it's been removed in favour of linking to the main Font Awesome upgrade guides.

I suggest an upgrade tip should be mentioned on the readme  to save time for other people upgrading.

**Comment by alecarneiro1993** [comment]
Indeed. After removing `font-awesome-sprockets`, everything started to work again. Glad I found this

**Comment by ericdfields** [comment]
Thank you! I've never added a fork with a branch to a Gemfile before. The syntax if needed is

```
gem 'resque-web', require: 'resque_web', git: 'https://github.com/EdCordata/resque-web', branch: 'fix/file-to-import-not-found-or-unreadable-font-awesome-sprockets'
```

**Comment by masatokawano** [comment]
Thank you. This helps.
